backend/backend.sh

```diff
@@ -1,55 +1,47 @@
 #!/bin/bash
-
-
-apt-get update
-
-
+apt-get update -y
 #need in swtpm 
-apt -y install autoconf-archive libcmocka0 libcmocka-dev procps iproute2 build-essential git pkg-config gcc libtool automake libssl-dev uthash-dev autoconf doxygen libjson-c-dev libini-config-dev libcurl4-openssl-dev libltdl-dev expect python3-yaml libglib2.0-dev dbus  libtasn1-6-dev libjson-glib-dev gawk socat libseccomp-dev uuid-dev udev libusb-1.0-0-dev libgnutls28-dev gnutls-bin
-
-
+apt  install -y autoconf-archive libcmocka0 libcmocka-dev procps iproute2 build-essential git pkg-config gcc libtool automake libssl-dev uthash-dev autoconf doxygen libjson-c-dev libini-config-dev libcurl4-openssl-dev libltdl-dev expect python3-yaml libglib2.0-dev dbus  libtasn1-6-dev libjson-glib-dev gawk socat libseccomp-dev uuid-dev udev libusb-1.0-0-dev libgnutls28-dev gnutls-bin
 #need in acs
-apt-get install -y libjson-c4  libjson-c-dev mariadb-server libmysqlclient-dev git make gcc wget 
-rm -rf /var/lib/apt/lists/*
-
+apt  install -y libjson-c4  libjson-c-dev mariadb-server libmysqlclient-dev git make gcc wget && rm -rf /var/lib/apt/lists
+# Remove existing directories and files
 rm -rf /root/tpm2  /root/acs  /root/startup.sh /root/swtpm
-
+# Change to the new directory and download ibmtss1.6.0.tar.gz
 mkdir /root/tpm2 && \
     cd /root/tpm2 && \
     wget https://versaweb.dl.sourceforge.net/project/ibmtpm20tss/ibmtss1.6.0.tar.gz && \
     tar zxvf ibmtss1.6.0.tar.gz
-
-# 多次尝试克隆 acs 代码库
+# Attempt to clone the acs repository up to 5 times with a 5 second sleep in between each attempt
 for i in {1..5}; do \
-    git clone https://github.com/kgoldman/acs.git /root/acs && \
+    [ ! -d "/root/acs" ] git clone https://github.com/kgoldman/acs.git /root/acs && \
```

Comment:
When multi commit in one pr, there is no need to submit a new commit to fix the problem commit. just use `git rebase ` to fix the problem commit.